lib/main.js

```diff
@@ -196,12 +196,10 @@ export default {
         if (version.major > 2) {
           // PHPCS v3 and up support this with STDIN files
           parameters.push(`--ignore=${ignorePatterns.join(',')}`);
-        } else {
+
           // We must determine this ourself for lower versions
```

Comment:
The only problem is the linter doesn't like an else statement if only an if statement inside it.